assets/theme-i18n.json

```
@@ -1,4 +1,5 @@
 {
+	"title": "Style variation title",
```

```suggestion
	"title": "Style variation name",
```

@oandregal This is how it's in the schema. Can you confirm that `Style variation name` is the desired context? If so, you'd need to change it here and also in the Behat test.

---

:+1: